docs/tasks/agent.md

```diff
@@ -218,7 +234,7 @@ The following information is copied directly from the included task in this role
   - --zone=public --permanent --add-port 50051/tcp
   - --reload
   when:
-    - ansible_distribution == "CentOS" or ansible_distribution == "RedHat"
+    - ansible_distribution == "CentOS" or ansible_distribution == "RedHat" or ansible_distribution == "SLES"
```

Comment:
Could this be Suse ?

Reply:
This information about managed hosts is provided by adhoc command.
For suse linux, ansible_distribution type is SLES.

Reply:
Hmm, this is what we define right ? We haven't called it RHEL or UEE , thought to maintain that consistency

Reply:
We are not defining them, these ansible facts are obtained from hosts. We just add condiitons based on values.